### DISCUSSION TITLE
Won't generate course over 500ac maps.

### DISCUSSION THREAD

**Discussion by fekim71** [Courseplay/Courseplay_FS22#2202] [Q&A]
I got everything up to date. Courseplay won't generate a course on custom fields over 500ac on new maps placed in the game. I have tried several maps and I can't get courseplay to generate a course. It will generate on a field that was built into the game but not on a new custom fields. It still works on a old map I'm playing. I can generate courses on fields over 2000ac. But not on any new maps. I let it render for over 15 minutes than I have to end task in task manager.

**Comment by Tensuko** [comment]
Start FS, try generate a field, when you stop it, attach logfile.

**Comment by fekim71** [comment]
Hers is the log file for FS22. Is there a Courseplay log file? If there is
, where is it?

**Reply by Tensuko** [reply]
Pls do not copy and paste the text of the log.
Attach it like this: (I did it for you now...)
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/10304837/log.txt)

**Reply by Tensuko** [reply]
And your log does not contain any information about a try of creating a course.

**Reply by Tensuko** [reply]
"I let it render for over 15 minutes than I have to end task in task manager."
Why you have to?
For 500AC or over 200HA depending on work with that can take some time and also depending on the field shape....

**Reply by fekim71** [reply]
I didn't copy and paste. I sent it as an attachment in Gmail.
 I've been rendering courses with fields bigger than 2000ac for almost a year on a couple of maps and till can. On the last few maps I tried to blank and use for flat starter maps Courseplay is useless. Rendering worked on all maps I tried before I deleted everything. Just can't figure way the courses wont render after deleting fields, trees, buildings and making terrain flat.

**Reply by Tensuko** [reply]
> I didn't copy and paste. I sent it as an attachment in Gmail. 

That doesn't work then

>  On the last few maps I tried to blank and use for flat starter maps Courseplay is useless.

Don't understand that at all...

> Rendering worked on all maps I tried before I deleted everything.

???

> Just can't figure way the courses wont render after deleting fields, trees, buildings and making terrain flat.

What are you actually trying to do....
Maybe Screenshots help.

**Reply by fekim71** [reply]
I'm trying to find or manipulate a map that is 16km x 16km or bigger. I want a blank flat map with only a vehicle spawning location.
 I can accomplish making the maps the why I want. But when I create a field with courseplay with draw a field tool, courseplay won't generate the course. Before I manipulated the maps with GE courseplay generated the courses just fine.
 I currently play with a 12km x 12km map that generates courses on fields as big as 3000ac in less than 2 minutes. But I cant seem to find a bigger starter(blank) map that will work with courseplay.

![map 12km 12km](https://user-images.githubusercontent.com/69436089/209979313-6727d566-e3da-410b-bde6-aff2f77da834.jpeg)

**Reply by Tensuko** [reply]
There might be limits to the system.
That could also be limited by the game engine.
